### DISCUSSION TITLE
Make the transform layer update by two or three frame

### DISCUSSION THREAD

**Discussion by Lioncub901** [keiwando/toonsquid#91] [Ideas]
I added a transform layer in my animation that have 2 frame length frames. Making the transform layers update every 1 frame makes the animation look not too good. Could you make it possible to update the transform layer every 1, 2, 3, etc. frames and not just 1.

**Comment by keiwando** [comment]
Just to clarify, you want to be able to add keyframes to the transform layer that are more than 2 frames apart and then between these keyframes have the property update only once every 2 frames?
Or are you also adding property keyframes every 2 frames and are just trying to get rid of the interpolation on that one frame in between? You can do the latter by selecting the "Hold" easing curve on the keyframes (though even in this case I guess it would still be more convenient if new keyframes maybe copied the easing curve from the previous keyframe instead of always defaulting to "Linear")

**Reply by Lioncub901** [reply]
What I am trying to say is that there should be an option(drop-down) to select how often(how many frames apart) key framing should update. It does not have to be only to the transformation layer but to the whole key framing system. This will make it possible to only animate in 1s, 2s, or 3s.
